imagepipeline/src/main/java/com/facebook/imagepipeline/core/ProducerSequenceFactory.java

```diff
@@ -102,6 +107,7 @@ public ProducerSequenceFactory(
       boolean useDownsamplingRatio,
       boolean useBitmapPrepareToDraw,
       boolean partialImageCachingEnabled) {
+    mContentResolver = context.getApplicationContext().getContentResolver();
```

Comment:
Could we pass the `ContentResolver` directly instead of the `Context`? Makes it harder to leak things.